### PR TITLE
ported to sbt 0.10.1

### PR DESCRIPTION
**Pull request by eed3si9n**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    log.info("Generating compatibility trait for Scala version  %s".format(v))
+    val body = if (v startsWith "2.9") {
+      """type CompatTraversable[A] = scala.collection.GenTraversableOnce[A]"""
+    } else {
+      """type CompatTraversable[A] = Traversable[A]"""
+    }    
+    val fullSource = """
+      package com.codecommit.antixml {
+        private[antixml] trait ScalaCompat {""" + body + """}
+      }
```
Comment (retronym) [inline build.sbt]: Should have '|' for stripMargin

File: build.sbt [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+scaladocOptions in Compile <++= (unmanagedSourceDirectories in Compile) map { (usd) =>
+  val scalaSrc: File = (usd filter { _.toString endsWith "scala" }).head
+  Seq(
+    "-sourcepath", scalaSrc.toString,
+    "-doc-source-url", "https://github.com/djspiewak/anti-xml/tree/master/src/main/scala€{FILE_PATH}.scala"
+  )
+}
+
+publishMavenStyle := true
```
Comment (retronym) [inline build.sbt]: That's the default, no?

Comment (djspiewak) [inline build.sbt]: Good question.  I would hope so…

File: build.sbt (hunk 1 above)
Comment (djspiewak) [inline build.sbt]: What do you mean?

Comment (eed3si9n) [inline build.sbt]: for some reason | didn't work. I am getting rid of stripMargin as 0.7.

File: build.sbt (hunk 2 above)
Comment (eed3si9n) [inline build.sbt]:
```
publishMavenStyle := true
```

is redundant because that's the default in 0.10. I'll remove it.

File: build.sbt (hunk 1 above)
Comment (retronym) [inline build.sbt]: The first '|' below is the REPL line continuation, I'm taking about the one in `|bar`. It's only aesthetically important.

```

scala>   """foo
     |      bar""".stripMargin
res0: String = 
foo
     bar

scala>   """foo
     |      |bar""".stripMargin
res1: String = 
foo
bar

```

Comment (eed3si9n) [inline build.sbt]: I missed `""" + body + """` in the middle. I pushed it back to `format(body)` and it strips fine now.

**Pull request merged**